modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -620,10 +620,18 @@
 			<param index="3" name="transfer_channel" type="int" default="0" />
 			<description>
 				Mark the following method for remote procedure calls. See [url=$DOCS_URL/tutorials/networking/high_level_multiplayer.html]High-level multiplayer[/url].
-				The accepted values:
-				- for [param mode] are [code]"any_peer"[/code] or [code]"authority"[/code];
-				- for [param sync] are [code]"call_remote"[/code] or [code]"call_local"[/code];
-				- and for [param transfer_mode] are [code]"unreliable"[/code], [code]"unreliable_ordered"[/code], or [code]"reliable"[/code].
+
+				If [param mode] is set as [code]"authority"[/code], only the authority peer can call this RPC function. Otherwise, any peer can call it and this parameter should be set to [code]"any_peer"[/code].
+				When configuring functions as RPC with [method Node.rpc_config], each of these mode respectively corresponds to the [constant MultiplayerAPI.RPC_MODE_AUTHORITY] and [constant MultiplayerAPI.RPC_MODE_ANY_PEER] rpc modes. See [enum MultiplayerAPI.RPCMode].
+				If a peer that is not the authority tries to call a function that is only allowed for the authority, this will print an error on both the local and the remote peers, and the function will not be executed.
```

Comment:
Each sentence here (and below) will make an entire paragraph. I don't think this was your intent? These should probably be grouped into one paragraph, so line breaks must go.